Examples/MotionManager/MotionManager/MotionManagerView.swift

```diff
@@ -2,6 +2,18 @@ import ComposableArchitecture
 import CoreMotion
 import SwiftUI
 
+private let readMe = """
+This demonstrates how to work with the MotionManager API from Apple's Motion framework.
```

Comment:
this project didn't have a read me, so adding one, and making it's focus on the mockability of dependencies in TCA